src/main/java/org/minions/devfund/marylin/WhackAMole.java

```diff
@@ -0,0 +1,165 @@
+        }
+        return false;
+    }
+
+    /**
+     * Method to whack a mole in the grid.
+     *
+     * @param x X axis in the grid.
+     * @param y Y axis in the grid.
+     */
+    private void whack(int x, int y) {
+        if (this.moleGrid[x][y] == 'M') {
+            this.moleGrid[x][y] = 'W';
+            this.score++;
+            this.molesLeft--;
+        }
+        this.attemptsLeft--;
+    }
+
+    /**
+     * Method to print the status of the game without shows the moles.
+     */
+    private void printGridToUser() {
+        for (char[] rowGrid:this.moleGrid) {
+            StringBuilder strBuilderRow = new StringBuilder();
+            for (char charItem:rowGrid) {
+                if (charItem == 'M') {
+                    strBuilderRow.append("*  ");
+                } else {
+                    strBuilderRow.append(charItem);
+                    strBuilderRow.append("  ");
+                }
+            }
+            LOGGER.info(strBuilderRow.toString());
+        }
+    }
+
+
+    /**
+     * Method to print the grid showing the status of the game even moles.
+     */
+    private void printGrid() {
+        for (char[] rowGrid:this.moleGrid) {
+            StringBuilder strBuilderRow = new StringBuilder();
+            for (char charItem:rowGrid) {
+                strBuilderRow.append(charItem);
+                strBuilderRow.append("  ");
+            }
+            LOGGER.info(strBuilderRow.toString());
+        }
+    }
+
+    /**
+     * Main method to play the game.
+     *
+     * @param args Default Argument.
+     */
+    public static void main(final String[] args) {
+        int numberOfMoles = 10;
+        int numAttemps = 50;
+        int gridLenght = 10;
+
+        //create a 10 by 10 grid
+        WhackAMole whackAMole = new WhackAMole(numAttemps, gridLenght);
+
+        //Place randomly a total of 10 moles
+        for (int i = 0; i < numberOfMoles; i++) {
+            Random rand = new Random();
+            int x = rand.nextInt(numberOfMoles);
+            int y = rand.nextInt(numberOfMoles);
+            if (!whackAMole.place(x, y)) {
+                System.out.println("There is already a mole at " + x + "," + y + ". Trying again!");
+                i--;
+            } else {
+                System.out.println("Mole places at " + x + "," + y);
+            }
+        }
+
+        //Try to whack 50 times the moles given the x and y
+        //Finish the game if we enter xAxis -1 and yAxis -1 printing the grid
+        //The game ends if the user is able to uncover all the moles or if the user runs out of attempts.
+
+        System.out.println("----------------------------");
+        whackAMole.printGrid();
+
+        System.out.println("-----You have a maximum of " + numAttemps + " attempts to get all the moles wacked-----");
+        int xAxis;
+        int yAxis;
+        Scanner xScanner;
+        Scanner yScanner;
+        for (int i = numAttemps; i > 0; i--) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reduce the total number of break and continue statements in this loop to use at most one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS135)